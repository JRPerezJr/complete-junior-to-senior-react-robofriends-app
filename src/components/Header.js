import React, { Component } from 'react';

export default class Header extends Component {
  render() {
    return <h1 className="f1">RoboFriends</h1>;
  }
}
