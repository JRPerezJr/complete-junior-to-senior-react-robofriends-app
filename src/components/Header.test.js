import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('<Header/> component tests', () => {
  it('sanity test', () => {
    expect(true).toBe(true);
  });

  it('renders Header component', () => {
    expect(shallow(<Header />)).toMatchSnapshot();
  });
});
