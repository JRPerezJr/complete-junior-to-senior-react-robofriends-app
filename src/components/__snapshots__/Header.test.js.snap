// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> component tests renders Header component 1`] = `ShallowWrapper {}`;
